test_runner/regress/test_timeline_size.py

```diff
@@ -936,6 +936,9 @@ def test_timeline_logical_size_task_priority(neon_env_builder: NeonEnvBuilder):
     tenant_id = env.initial_tenant
     timeline_id = env.initial_timeline
 
+    # just make sure this doesn't hit an assertion
+    client.timeline_detail(tenant_id, timeline_id, force_await_initial_logical_size=True)
```

Comment:
without the first commit, this would fail as:

```
FAILED test_runner/regress/test_timeline_size.py::test_timeline_logical_size_task_priority[debug-pg16] - fixtures.pageserver.http.PageserverApiException: HTTP request handler task panicked
```

(and have an allowed_error check fail as well).